doc/reference/annotations.rst

```diff
@@ -917,3 +930,13 @@ Example:
             return 1;
         }
     ...
+
+Enum support
+~~~~~~~~~~~~
+
+Enums are automatically detected using typed properties typehints.
```

Comment:
It looks like it is not added to `SerializerBuilder::addDefaultHandlers()` so it needs to be added manually? 
Is it possible how many people adopted Serializer at PHP8.1 already? I wonder if it can be a breaking change for them.

Reply:
fixed